base/operators.jl

```diff
@@ -827,17 +827,28 @@ julia> [1:5;] |> x->x.^2 |> sum |> inv
 Compose functions: i.e. `(f ∘ g)(args...)` means `f(g(args...))`. The `∘` symbol can be
 entered in the Julia REPL (and most editors, appropriately configured) by typing `\\circ<tab>`.
 
+Function composition also works in prefix form: `∘(f,g)` is the same as `f ∘ g`.
+The prefix form also supports multiple function composition: `∘(f,g,h) = f ∘ g ∘ h`
+and splatting for an iterable with functions `∘(fs...)`
```

Comment:
The sentence should finish with a period, I guess.

Reply:
Fixed.